rules/windows/defense_evasion_microsoft_defender_tampering.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/11/22"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when one or more features on Microsoft Defender is disabled.
+"""
+from = "now-9m"
```

Comment:
It needs a false_positives section, as this will look for the registry changes, will be very noisy because it covers GUI and GPOs. These are (mostly) legitimate used by administrators.